### PR TITLE
Push to dockerhub on merge to master & readd integration tests

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,17 +7,16 @@ services:
...
       name: build test
       install: true
       script: BUILD_PHASE=builder make docker_build
     - if: fork = false
       stage: test
-      name: build
+      name: build, integration test, and push
       install: true
-      script: BUILD_PHASE=builder make docker_build
+      script: BUILD_PHASE=builder make docker_build && make k8s_integration && make dockerhub_push
```
Comment (wild-endeavor) [inline .travis.yml]: but we'll get rid of this right?

Comment (katrogan) [reply .travis.yml]: no, why?

Comment (wild-endeavor) [reply .travis.yml]: i thought we didn't want to push to dockerhub on PRs

Comment (katrogan) [reply .travis.yml]: this is if `fork = false`

Comment (wild-endeavor) [reply .travis.yml]: oh that's what that does?  cool

Comment (katrogan) [reply .travis.yml]: ugh forget it, i'll figure out creds to push in another pr

**Pull request merged**